lightning/src/ln/functional_test_utils.rs

```diff
@@ -763,6 +763,19 @@ macro_rules! check_closed_broadcast {
 	}}
 }
 
+/// Check that a channel's closing channel event has been issued
+#[macro_export]
+macro_rules! check_closed_event {
```

Comment:
This needs to check the reason, at least, if not also the contents of the reason variant if applicable.